### DISCUSSION TITLE
Website looking weird? Anyone else?

### DISCUSSION THREAD

**Discussion by isaac-souza** [twbs/bootstrap#36103] [General] [answered]
It has been like this to me for weeks.

Anyone else?

![Screenshot from 2022-04-01 12-37-19](https://user-images.githubusercontent.com/35014851/161296330-205b1a18-669f-483c-add0-e653f0b093b2.png)

![Screenshot from 2022-04-01 12-40-00](https://user-images.githubusercontent.com/35014851/161296625-77292fdf-8823-47aa-8936-12bdee08a0b6.png)

![Screenshot from 2022-04-01 12-40-47](https://user-images.githubusercontent.com/35014851/161296749-de500ebc-7a99-48e9-9410-38f7dc622ef2.png)

**Comment by mdo** [comment, marked as answer]
Likely a caching issue or you've zoomed in too far hah.
